addons/takistan/CfgVehicles_TNA.hpp

```diff
@@ -0,0 +1,352 @@
+// Units - Takistan National Army
+class TACU_Takistan_U_TNA_Major: TACU_Main_U_INDEP_Soldier_Base {
+    author = "TyroneMF";
+    displayName = "Major";
+    faction = "TACU_Takistan_TNA";
+    scope = 2;
+    scopeCurator = 2;
+    identityTypes[] = {"LanguagePER_F", "Head_TK", "NoGlasses"};
+    genericNames = "TakistaniMen";
+    icon = "iconMan";
+    role = "Rifleman";
+    uniformClass = "CUP_U_O_TK_Officer";
+    backpack = "";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_Major.jpg);
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_O_TK_OfficerBelt", "CUP_H_TK_Beret"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_O_TK_OfficerBelt", "CUP_H_TK_Beret"};
+    Items[] = {mag_5("ACE_fieldDressing")};
+    respawnItems[] = {mag_5("ACE_fieldDressing")};
+    weapons[] = {"CUP_hgun_M9", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_hgun_M9", "Throw", "Put"};
+    magazines[] = {mag_4("CUP_15Rnd_9x19_M9")};
+    respawnMagazines[] = {mag_4("CUP_15Rnd_9x19_M9")};
+    editorSubcategory = "TACU_Takistan_EdSubCat_Officer";
+};
+
+class TACU_Takistan_U_TNA_Lieutenant: TACU_Takistan_U_TNA_Major {
+    displayName = "Lieutenant";
+    uniformClass = "CUP_U_O_TK_Green";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_Lieutenant.jpg);
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_RRV_TL", "CUP_H_TK_Beret"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_RRV_TL", "CUP_H_TK_Beret"};
+    weapons[] = {"CUP_arifle_M16A2", "CUP_hgun_M9", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_arifle_M16A2", "CUP_hgun_M9", "Throw", "Put"};
+    magazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9")};
+    respawnMagazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_3("CUP_15Rnd_9x19_M9")};
+};
+
+class TACU_Takistan_U_TNA_NCO: TACU_Takistan_U_TNA_Lieutenant {
+    displayName = "NCO";
+    editorPreview = QPATHTOF(ui\Takistan_U_TNA_NCO.jpg);
+    linkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_PASGT_no_bags_OD", "CUP_H_Ger_M92_RGR"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS_RADIO, "CUP_V_B_PASGT_no_bags_OD", "CUP_H_Ger_M92_RGR"};
+    weapons[] = {"CUP_arifle_M16A2", "CUP_hgun_M9", "Throw", "Put"};
+    respawnWeapons[] = {"CUP_arifle_M16A2", "CUP_hgun_M9", "Throw", "Put"};
+    magazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_2("CUP_15Rnd_9x19_M9"), mag_2("SmokeShell")};
+    respawnMagazines[] = {mag_8("CUP_30Rnd_556x45_Stanag"), mag_2("CUP_15Rnd_9x19_M9"), mag_2("SmokeShell")};
```

Comment:
Amount of mags unsupported by current uniform + vest combo.
Currently there's room for 7 rifle mags, 0 pistol mags and 1 smoke. (Keep in mind 1 rifle and 1 pistol mag is loaded, so that would technically be `mag_8` for rifle and `mag_1` for pistol).

If you set the rifle mag to be `mag_6`, then the pistol and smokes will fit.

Reply:
Dropped all units to max of 6 mags and 2 handgun mags.

Units with only handguns are unaffected.